dnf5/config/usr/lib/dnf5/aliases.d/compatibility.conf

```diff
@@ -0,0 +1,125 @@
+########################################
+# Global options compatibility aliases
+########################################
```

Comment:
I can see there still are a couple of commands specific options compatibility aliases. Namely the advisory shared options: `--advisory`, `--bz` and `--cve`. 
These are per command aliases so `upgrade`, `repoquery` and `install` all have them, and `advisory` command also has two.

I think these would be possible to define with something like:
```
['repoquery.advisory']
type = 'cloned_named_arg'
long_name = 'advisory'
source = 'repoquery.advisories'
complete = false
```
but it would be copied for each command - 4 times. 
Do we plan to do this as well? Then the code base wouldn't have any hard-coded aliases.

Reply:
Yes. My idea is source code without hard-coded aliases. A few aliases left in the source code, this PR is already so big, but I'll look at it again.

Reply:
It is not a problem, it can be done in the next PR.

I will merge once the CI finishes.